doc/manpage-newsboat.asciidoc

```diff
@@ -4,7 +4,7 @@ Alexander Batischev <eual.jp@gmail.com>
 
 == NAME
 
-newsboat - an RSS/Atom feed reader for text terminals
+Newsboat - an RSS/Atom feed reader for text terminals
```

Comment:
This is a manpage, and this line describes a command. The rest of the document talks of the program. Confusing, I know, but that's how it is :) Please revert this change.